### ISSUE TITLE
[Screen Reader - Microsoft Form Pro - Header] Incorrect role defined for 'Environment' combo box.

### ISSUE THREAD

**Issue by sonimishra** [OfficeDev/office-ui-fabric-react#11320]
### Environment Information
#URL:https://forms.office-int.com/
Version 1903 (OS Build 19018.1)

### Describe the issue:

User Experience:
Screen reader user will get confused with the control type of the control if role is not correctly defined due to this screen reader user will not able to access the control efficiently.

Note: User credentials should NOT be included in the bug.

Repro Steps:
1. Open URL: https://forms.office-int.com/ and login using valid credentials.
2. Navigate on the header elements of the homepage.
3. Verify the issue.
 

### Please provide a reproduction of the issue in a codepen:

-------------------

#### Actual behavior:
Incorrect role defined for 'Environment' i.e - role='listbox'.

#### Expected behavior:
Correct role should be defined for 'Environment'. It should be defined as 'Combo box'.

**Comment by khmakoto** [comment]
Hi @sonimishra , I think what you're logging is an application bug that needs to be investigated by someone from the Office Forms team since it is not clear if this is indeed a Fabric bug. Please contact that team first and, if it is indeed a Fabric issue then we can start working on it. Thanks!

**Comment by sonimishra** [comment]
Hi @khmakoto, We have discussed this bug with Forms Pro team and according to them 'Environment' Picker control on header section is manage by Office UI fabric team.

**Comment by khmakoto** [comment]
@sonimishra, could you provide a screenshot of the control that is causing the problem then? The description is very vague and I don't really know what you control you're referring to when I log in and look at the page.

**Comment by sonimishra** [comment]
Hi @khmakoto screenshot is attached
![MAS4 1 2_Incorrect role defined](https://user-images.githubusercontent.com/30456910/70115774-c08b3a00-1686-11ea-9d75-3b92cc1b3a4c.png)

**Comment by khmakoto** [comment]
Hi @sonimishra, I can see the control now, however, you're using a `Dropdown` control, not a `ComboBox` control, there's a difference in both anatomy and functionality between both and they have different accessibility roles because of this.

Given that, I don't think this is a problem with Fabric controls, since we are applying the correct role to the correct component. Here are my the two actions I could see if this is really a problem for you moving forward:
1. Use the `ComboBox` component in Fabric instead of the `Dropdown` control.
2. Have the application override the `role` attribute of the `Dropdown` control and handle accessilibyt related concerns while doing that themselves, since for the purposes of the `Dropdown` component we're using the correct thing.

Let me know if there is anything else I can help with or anything I can clarify. Thanks!
